test/getHashDigest.test.js

```diff
@@ -34,7 +34,8 @@ describe("getHashDigest()", () => {
     ],
     ["test string", "md5", "base52", undefined, "dJnldHSAutqUacjgfBQGLQx"],
     ["test string", "md5", "base64", undefined, "b421md6Yb6t6IWJbeRZYnA=="],
-    ["test string", "md5", "base26", 6, "bhtsgu"],
+    ["test string", "md5", "base26", undefined, "bhtsgujtzvmjtgtzlqvubqggbvgx"],
+    ["test string", "md5", "base26", 6, "ggbvgx"],
```

Comment:
Strings are now sliced from the end to get more uniformly distributed chars.

Previously:
```
bhtsgujtzvmjtgtzlqvubqggbvgx
bhtsgu
```

Now:

```
bhtsgujtzvmjtgtzlqvubqggbvgx
                      ggbvgx
```

@alexander-akait